scipy/optimize/_linprog_highs.py

```diff
@@ -236,6 +240,20 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
                     function with respect to the lower and upper
                     `bounds`.
 
+            mip_node_count : int
+                Only present when `integrality` is not `None`. The number of
+                subproblems or "nodes" solved by the MILP solver.
```

Comment:
```suggestion
                The number of subproblems or "nodes" solved by the MILP
                solver. Only present when `integrality` is not `None`.
```
I think we should lead with the description and follow with the details.

Reply:
Done